src/managers.py

```diff
@@ -0,0 +1,16 @@
+from src.models import Department, SessionLocal
+
+
+class DepartmentManager:
+    @staticmethod
+    def insert_data(data: dict):
+        with SessionLocal() as session:
+            session.add(Department(**data))
+
+    def get_info():
+        with SessionLocal() as session:
+            return session.query(Department).all()
+
+    def get_id(dep_id):
+        with SessionLocal() as session:
+            return session.query(Department).filter_by(id=dep_id).first()
```

Comment:
Let's refactor naming a little bit: `insert_data` rename to `create`, `get_info` - `get_all`, `get_id` - `get_by_id`, and `dep_id` let's rename to `department_id`. Don't make such cuts, better to use full names